### ISSUE TITLE
Instructions to configuration

### ISSUE THREAD

**Issue by brunocalado** [luvolondon/quickstart#1]
Hi. What a great idea.

I can't find a way to customize. 

Can you provide a little more instrucions?

Thank you.

**Comment by luvolondon** [comment]
Hi,
are you thinking about the look of the login page or the other tool with the steps for game setup? 
In any case this is all just started and still very fresh. I´m collecting ideas what would help a gm to get new players involved and hyped for the game. But also have a smooth start without overwhelming them with a complex ui.

**Comment by brunocalado** [comment]
It's perfect the idea.

I just want to config the information for each button. I would like to translate it too.

By game setup do you mean configurate each button content? I want to do this.

![image](https://user-images.githubusercontent.com/4999783/99280199-d64e2c80-280f-11eb-956d-d8e4d8f88c1a.png)

**Comment by luvolondon** [comment]
Yeah, that is the part I just started with. I´m currently looking at Javascript libraries to embed slideshows with animations, images, text etc. into the game. I´m currently looking at impress.js (https://impress.js.org), which does some amazing animated content presentations. You could then use a GUI tool like http://www.impressed.co/ to build your slideshow, export it and upload it into this module. 

Regarding translation: The module already has lang files for the direct-login-page, the same mechanism could be used for this part of the module.

Have a look at the next releases. It´ll hopefully show more of the editing features.

**Comment by luvolondon** [comment]
FYI: I´ve added the presentation library revealjs (https://revealjs.com) to show slideshows. You can have a look at templates/slideshow.html and the revealjs doc to see how slideshows will be setup later. When in step "Slideshow" for now the GM can use the arrow keys to switch between to slides and this will be auto-synced to the player´s screens. 

The steps "Setting" and "Characters" will present a shared creation of the setting based on story cards dealt to the players. This is still very much work-in-progress.

**Comment by brunocalado** [comment]
This is one of the greatest idea. It can make getting start with Foundry VTT smoother.

Thank you.

**Comment by luvolondon** [comment]
Hi, I´ve created a new release. You can now setup a slideshow, show it to your players and have notes for each slide in the GM view.

**Comment by brunocalado** [comment]
Hi.

I'm testing. It looks great. The overview mode is working great.

Is it possible to set the bar on right to hide when in Slide show mode?
![image](https://user-images.githubusercontent.com/4999783/100026449-923bc880-2dc9-11eb-80fe-6dcb6b6f248f.png)

I can't make the sound to play. I tried
`<section data-audio-src="A Plinian Eruption" data-journalentry="Imagem1">`
and
`<section data-audio-src="Adventure Music Battle" data-journalentry="Imagem2">`
For 
![image](https://user-images.githubusercontent.com/4999783/100026891-5fde9b00-2dca-11eb-8288-4a4838945491.png)

**Comment by brunocalado** [comment]
After active the module I also get this warning stuck.
![image](https://user-images.githubusercontent.com/4999783/100027079-cc599a00-2dca-11eb-891f-1b00fccd0b33.png)

Console
`DevTools failed to load SourceMap: Could not load content for https://mywebpage:30000/modules/quickstart/revealjs/reveal.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE`

**Comment by luvolondon** [comment]
Hi, thank you for your feedback. 
Hiding the right bar when a slideshow starts for all players is good idea.
"Adventure Music Battle" should work, you can only give names of playlists, not individual entries. Try only using that name.
The SourceMap warning should not be a problem, I´ll comment that out from reveal.js library.

Just a quick note: Don´t put too much work in setting up your slideshows. I´m currently working on a way to edit slideshows directly in-game as special JournalEntries with a graph editor library. Hope to get something running soon.

**Comment by brunocalado** [comment]
> lEntries with a graph editor library.
"Don´t put too much work in setting up your slideshows" Thank for this tip. lol

Ok. Thank you.

**Comment by luvolondon** [comment]
FYI: I´ve moved the slideshow functionality to an extra module "slideshow". With this module you can create standalone slideshows as JournalEntries and start playback from the editor. The editor for a slideshow allows for interactive creation of the slides with drag-n-drop of playlist and journalentries for content. 
In the next step I´ll change this module to use the functionality of the slideshow module.

**Comment by brunocalado** [comment]
Amazing
